graviti/portex/field.py

```diff
@@ -49,6 +50,14 @@ def _check_value(cls, value: PortexType) -> None:
         if not isinstance(value, PortexType):
             raise TypeError(f'The value in "{cls.__name__}" should be a PortexType')
 
+    @classmethod
+    def _check_key(cls, key: str) -> None:
+        super()._check_key(key)
+        if re.search("[A-Z]", key):
```

Comment:
can we use `not str.isupper` rather than regex?

Reply:
`not str.isupper` is True when not all letters in the string are uppercase. It does not distinguish cases like `abC` and `abc`.